src/components/resume-items/education-item.tsx

```diff
@@ -0,0 +1,131 @@
+  const [major, setMajor] = useState("");
+  const [minor, setMinor] = useState("");
+  const [description, setDescription] = useState("");
+
+  const handleFormSubmit = async (event: any) => {
+    event.preventDefault();
+
+    const educationData = {
+      universityName,
+      startDate,
+      endDate,
+      major,
+      minor,
+      description,
+    };
+
+    console.log(educationData);
+
+    // API call to save data (replace placeholder with your actual implementation)
+    try {
+      const response = await fetch("/api/save-education-data", {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify(educationData),
+      });
+
+      if (response.ok) {
+        // Handle successful submission (e.g., close dialog, clear inputs, display feedback)
+      } else {
+        // Handle error from API
+      }
+    } catch (error) {
+      console.error("Error submitting form:", error);
+    }
+  };
+
+  return (
+    <Dialog>
+      <DialogTrigger asChild>
+        <Button className="text-left h-full w-full" variant="ghost">
+          Education
+        </Button>
+      </DialogTrigger>
+      <DialogContent className="sm:max-w-[600px]">
+        <DialogHeader>
+          <DialogTitle>Add Education</DialogTitle>
+          <DialogDescription>
+            Fill in the following information
+          </DialogDescription>
+        </DialogHeader>
+        <form onSubmit={handleFormSubmit}>
+          <div className="grid grid-cols-2 gap-4 flex">
+            <Input
+              className="w-[300px]"
+              id="item-name"
+              placeholder="University Name"
+              value={universityName}
+              onChange={(e) => setUniversityName(e.target.value)}
+            />
+            <div className="flex justify-end">
+              <Input
+                className="w-[90px] mr-[15px]"
+                id="start-date"
+                placeholder="Start Date"
+                value={startDate}
+                onChange={(e) => setStartDate(e.target.value)}
+              />
+              <Input
+                className="w-[90px]"
+                id="end-date"
+                placeholder="End Date"
+                value={endDate}
+                onChange={(e) => setEndDate(e.target.value)}
+              ></Input>
+            </div>
+            <Input
+              className="w-[300px]"
+              id="major"
+              placeholder="Select Major"
+              value={major}
+              onChange={(e) => setMajor(e.target.value)}
+            ></Input>
+            <div className="flex justify-end">
+              <Input
+                className="w-[195px]"
+                id="Minor"
+                placeholder="Minor (Optional)"
+                value={minor}
+                onChange={(e) => setMinor(e.target.value)}
+              ></Input>
+            </div>
+            <div className="w-[550px]">
+              <Textarea
```

Comment:
I'm thinking about how we are going to have to handle bullet points, since each bullet needs to be split out into its own part for the latex rendering. The best option might be to have the user input each bullet as its own input (see image). The other option is markdown (which I think we will need to use anyway for bold/italics) but the issue with translating markdown to latex is that markdown allows nested lists and it might create bugs.
![Screenshot 2024-02-26 135836](https://github.com/SprintCV-S24/frontend-SprintCV-S24/assets/114269804/90b823aa-132b-4ee1-82e7-17c46ec388a1)